src/main/java/frc/robot/Constants.java

```diff
@@ -35,10 +53,126 @@ public static class Ports {
     }
 
     public static class CustomUnits {
-        public static final AngleUnit TalonEncoderCounts = derive(Rotations).splitInto(2048).named("Talon Encoder Counts").make();
         public static final PerUnit<DistanceUnit, AngleUnit> MetersPerRotation = Meters.per(Rotations);
     }
 
+    public static final class Swerve {
+        public static final int pigeonID = 13;
+        public static final boolean REDUCE_SPEED = false;
```

Comment:
set to true for programming, we should only change this back once we are doing more intense testing